ci/pod/docker-compose.first.yml

```diff
@@ -46,6 +46,15 @@ services:
     networks:
       consul_net:
 
+  consul_3:
+    image: hashicorp/consul:1.16.2
```

Comment:
Why not use consul:1.7 like above example?

Reply:
Tests aren't passing using consul 1.7 because i think it has a bug. It doesn't bootstrap the ACL system using my predefined token and always creates a new random one. It's also not recommended anymore to use the docker images without the *hashicorp/* prefix.